server/src/domain/metadata/metadata.service.ts

```diff
@@ -81,10 +83,13 @@ export class MetadataService {
   ) {
     this.configCore = SystemConfigCore.create(configRepository);
     this.storageCore = StorageCore.create(assetRepository, moveRepository, personRepository, storageRepository);
-    this.configCore.config$.subscribe(() => this.init());
   }
 
   async init(deleteCache = false) {
+    if (!this.subscription) {
```

Comment:
Can you explain a bit about this piece of logic here?

Reply:
Rxjs has a way to subscribe to events and unsubscribe and they encapsulate that in a subscription. I'm unsubscribing here to make sure everything is cleaned up